tools/database/src/commands.rs

```diff
@@ -52,6 +52,9 @@ enum SubCommand {
 
     /// Loads an in-memory trie for research purposes.
     LoadMemTrie(LoadMemTrieCommand),
+    /// Splits given shard on a given boundary account and prints child sizes
```

Comment:
maybe let's be more explicit what is computed, as we have a lot of ways to do so
```suggestion
    /// Splits given shard on a given boundary account and prints RAM usage of the child sizes
```